composer.json

```diff
@@ -14,7 +14,7 @@
     "ext-json": "*",
     "ext-libxml": "*",
     "ext-spl": "*",
-    "ampproject/amp-toolbox": "dev-main",
+    "ampproject/amp-toolbox": "dev-add/55-optimize-hero-images-transformer",
```

Comment:
I assume tests will begin to fail once https://github.com/ampproject/amp-toolbox-php/pull/262 is merged and the branch is deleted.

Reply:
Yeah. Once the PR is merged, I'll update this to use `dev-main` or else 0.6.0.